example/plugins/backends/spidsaml2_backend.yaml

```diff
@@ -124,7 +124,7 @@ config:
 
         # this instantiate the attribute_consuming_service
         required_attributes: ['spidCode', 'name', 'familyName', 'fiscalNumber', 'email']
-        requested_attributes: !ENV SATOSA_REQUESTED_ATTRIBUTES
+        requested_attributes: !ENV SATOSA_REQUESTED_SPID_ATTRIBUTES
```

Comment:
could you please set default values along with this ENV? (eg, :-default ...)

could you please provide in this PR the CIE attribute-maps in the example folder and use a default parameter that points there?

Reply:
@MdreW ^